andela/ah-frontend-athena
# #162163244 List bookmarked articles

#### What does this PR do?
Enable an author's haven user view a list of their bookmarked articles.

#### Description of tasks to be completed?
Create the necessary files (actions, reducers, components) to fetch the data from the endpoint.
Transfer the data retrieved from the endpoint, to the presentational component and render the list of bookmarked articles.
Write tests for this functionality.

#### How can this be manually tested?
- Try to get bookmarks without logging in (navigate to `/bookmarks`) to see the response given
- Log in with new credentials (a user without bookmarks) to see the response given.
- Log into the application using the credentials (and navigate to `/bookmarks`):
`email: andrew.twijukye@andela.com`
`password: #Yurizahard123`
(this user has some bookmarks)


#### Relevant pivotal tracker story?
162163244

#### Screenshots
No bookmarks
![image](https://user-images.githubusercontent.com/5712103/51955786-26e77080-2456-11e9-9326-314bcc0881bf.png)

Not logged in
![image](https://user-images.githubusercontent.com/5712103/51955836-5eeeb380-2456-11e9-9aa5-b3e29e54938a.png)

Logged in & have bookmarks
![image](https://user-images.githubusercontent.com/5712103/52047479-1e785e00-255a-11e9-90f3-2e17b080f386.png)
